api/apps/api/test/e2e.config.ts

```diff
@@ -118,7 +119,7 @@ export const E2E_CONFIG: {
       minimal: () => ({
         name: faker.random.words(5),
         organizationId: faker.random.uuid(),
-        planningUnitGridShape: PlanningUnitGridShape.Hexagon,
+        planningUnitGridShape: PlanningUnitGridShape.Irregular,
```

Comment:
🤔 even if this _happens not to interfere_ with other tests, I recommend to create a distinct fixture for the use we're making in the new tests in this PR. `PlanningUnitGridShape.Irregular` is not even supposed to be used anywhere in the platform for the time being, so it would be at least confusing to have this as a default for this fixture.

I'd either create a sibling fixture such as `projects.valid.minimalWithIrregularGridShape` or use the default `projects.valid.minimal` in the new test with a spread to overwrite the `planningUnitGridShape` prop with the desired `PlanningUnitGridShape.Irregular` value

Reply:
actually, looking at the new test, I'm not even sure about the need for the change I'm commenting about for that test... so anyway - wouldn't keeping `planningUnitGridShape: PlanningUnitGridShape.Hexagon` work here?

Reply:
As `Hexagon` is a regular shape, we would need to add a `planningUnitId` or any of the GADM id so the controller doesn't return a 400. Maybe adding a pu id is enought?

Reply:
ah ok, I see - yes, the main issue here may be with `createWorld.WhenCreatingProjectWithoutAdminAreas()` in `api/apps/api/test/project-planning-units/world.ts`, which is used in `api/apps/api/test/project-planning-units/planning-units.e2e-spec.ts` because this will fail before even getting to the assertion.

we also have one occurrence in `projects.e2e-spec.ts` and one in `organizations.e2e-spec.ts` where we should be ok with using `minimalInGivenAdminArea()` instead.

in these cases, using `Irregular` as grid shape would be cheating a bit, and the current behaviour (accepting a project without admin area nor planning area id) is incorrect, so it's best to remove the use of `projects.valid.minimal` instead.